armi/materials/uraniumOxide.py

```diff
@@ -131,9 +131,7 @@ def applyInputParams(
         material.FuelMaterial.applyInputParams(self, *args, **kwargs)
 
     def setDefaultMassFracs(self) -> None:
-        r"""
-        UO2 mass fractions. Using Natural Uranium without U234.
-        """
+        r"""UO2 mass fractions. Using Natural Uranium without U234."""
```

Comment:
Interesting that `ruff` adds `r` when needed but doesn't remove it when it's not.

Not asking you to comb through these, just making an observation :-)

Reply:
So, I BELIEVE there is a different `ruff` rule for removing the unnecessary r's.  But that's just not part of this PR.

Soon!

Reply:
<3